Dockerfile

```diff
@@ -1,7 +1,7 @@
 ARG RISK=stable
 ARG UBUNTU=20.04
-ARG http_proxy=""
-ARG https_proxy=""
+#ARG http_proxy=""
+#ARG https_proxy=""
```

Comment:
Could these be removed instead of commented out?

Reply:
Done!